Gems/AtomLyIntegration/CommonFeatures/Code/Source/SharedPreview/SharedPreviewContent.h

```diff
@@ -51,7 +51,7 @@ namespace AZ
             static constexpr float NearDist = 0.001f;
             static constexpr float FarDist = 100.0f;
             static constexpr float FieldOfView = Constants::HalfPi;
-            static constexpr float CameraRotationAngle = Constants::QuarterPi / 2.0f;
+            static constexpr float CameraRotationAngle = Constants::QuarterPi / 3.0f;
```

Comment:
might be just better to define it as a const, because this random fraction doesn't make sense at this point